O/OCCT/build_tarballs.jl

```diff
@@ -0,0 +1,115 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "OCCT"
+version = v"7.6.2"
+
+# Collection of sources required to build Open CASCADE Technology (OCCT)
+sources = [
+    GitSource("https://github.com/Open-Cascade-SAS/OCCT.git",
+              "bb368e271e24f63078129283148ce83db6b9670a"), # V7_6_2 @ April 29, 2022
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/OCCT
+if [[ ${target} == *musl* ]]; then
+    atomic_patch -p1 "${WORKSPACE}/srcdir/patches/OSD_MemInfo.cxx.patch"
+    atomic_patch -p1 "${WORKSPACE}/srcdir/patches/OSD_signal.cxx.patch"
+    atomic_patch -p1 "${WORKSPACE}/srcdir/patches/Standard_StackTrace.cxx.patch"
+fi
+if [[ ${target} == *freebsd* ]]; then
+    atomic_patch -p1 "${WORKSPACE}/srcdir/patches/Standard_CString.cxx.patch"
+    atomic_patch -p1 "${WORKSPACE}/srcdir/patches/STEPConstruct_AP203Context.cxx.patch"
+fi
+mkdir build
+cd build
+cmake -Wno-dev .. \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_LIBRARY_TYPE=Shared \
+    -DBUILD_MODULE_Draw=0 \
+    -DBUILD_MODULE_Visualization=0 \
+    -DBUILD_MODULE_ApplicationFramework=0
+make -j${nproc}
+make install
+if [[ ${target} == *mingw* ]]; then
+    # Correct installation paths for windows
+    ln -s ${prefix}/win*/gcc/lib/* ${prefix}/lib
+    ln -s ${prefix}/win*/gcc/bin/* ${prefix}/bin
```

Comment:
Could maybe move instead of creating links (which become copies anyway in the final tarball)?

Reply:
Is this safe to just move the `.dll` outputs? I could also patch the `CMakeLists.txt` for Windows builds in order to install to just `${prefix}/bin` and `${prefix}/lib` if that would be preferred.

Reply:
> Is this safe to just move the .dll outputs?

I have no idea about the specific case, I don't know the package you're building here.  There could be a problem if there are somehow references to relative paths, but they'd break anyway as it is now: when you create a symlink for windows we automatically make a copy when we create the tarball, as Windows doesn't support very well symlinks.  So either way we get into exactly the same state.

> I could also patch the CMakeLists.txt for Windows builds in order to install to just ${prefix}/bin and ${prefix}/lib if that would be preferred.

Note that both executables and shared libraries for windows should be under `${prefix}/bin`, even though it's always better to use the appropriate environment variables `${bindir}` and `${libdir}`: https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables

Reply:
Agreed, this is resolved by patching the install directory path for Windows systems in 612be31 so no move/copy of the libraries is required.